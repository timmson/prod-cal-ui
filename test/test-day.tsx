import React from "react"
import {render, screen} from "@testing-library/react"

import Day from "../src/day"

describe("Day should", () => {

	test("render", () => {
		const d = {type: "type", date: 1}

		render(<table>
			<tbody>
				<tr>
					<Day day={d}/>
				</tr>
			</tbody>
		</table>
		)

		//expect(screen.getByText("1")).toBeInDocument()
		expect(screen.getByText("1")).toBeDefined()
	})

})
