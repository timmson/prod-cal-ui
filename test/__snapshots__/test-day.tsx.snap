// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Day should be equal to snapshot 1`] = `
<td
  className="type"
>
  1
</td>
`;
